### ISSUE TITLE
Unexpected behaviour with updates --patch

### ISSUE THREAD

**Issue by JamieMason** [silverwind/updates#24]
Hi, thanks for this great project. 

I found something which to me looks like it could be a bug and I thought I should show you. When running `updates --patch` I received this entry:

```
styled-components  2.5.0-1  4.2.0  https://github.com/styled-components/styled-components
```

See what you think but, looking at the output of `npm view styled-components versions`, I would expect that no patch updates are available for styled-components@2.5.0-1 as the next highest version after 2.5.0-1 is 3.0.1.

EDIT: Just noticed this also happens with `updates --minor`

Thank you for your time.

**Comment by silverwind** [comment]
Yes this is almost certainly a bug, I think the fact that you're on a prerelease tag confuses it, will check.

**Comment by silverwind** [comment]
Fixed in 8.0.2.

**Comment by JamieMason** [comment]
Thanks @silverwind 🤝
